contrib/sip_proxy/filters/network/source/utility.h

```diff
@@ -261,6 +261,7 @@ template <typename T, typename K, typename V> class CacheManager {
 
   /**
    * NOTE: This may throw exception std::out_of_range if type or key doesn't exist.
+   * use contains(type, key) to check exist first.
    */
```

Comment:
Rework to always use find. Never us at.

Reply:
@mattklein123 Thanks for your comments.

Firstly I want to implement a `find` method like what does as `std::map`, but I can't figure out the solution because different iterator maybe returned in a nested map, so write a method called `at` which return empty value if not exist.
```
[[ret_type]] find(const T& type, const K& key) {
  auto it = caches_.find(type);
  if (it != caches_.cend()) {
    return it->second.find(key);    // iterator of Cache<K, V>::iterator
   } else {
    return [[ the iterator of caches_ ]]
  }
}
```

Reply:
drive-by comment: if your goal is to return an optional value if it's in one of several collections, you could return absl::optional<V>, or OptRef<V> (see envoy/common/optref.h) if you want V to be a reference.

Reply:
Thanks @jmarantz , I already aware of this, but I don't want to make thing more complex.